admin/views/pupil-pin/generate-pins-overview.ejs

```diff
@@ -8,6 +8,16 @@
 </div>
 <div class="grid-row">
     <div class="column-two-thirds">
+        <% if (error) { %>
+        <div class="error-summary" role="group" aria-labelledby="error-summary-heading-1">
+            <h2 class="heading-medium error-summary-heading" id="error-summary-heading-1">
+                Unable to proceed. Error: <%= error %>
```

Comment:
Hard-coding this `Unable to proceed` part of the message seems brittle.  We should have consistent error handling in the front-end.

Reply:
@jon-shipley I'll amend to include the common part in the errors